Sources/Runestone/TextView/Core/TextView.swift

```diff
@@ -568,6 +568,8 @@ open class TextView: UIScrollView {
             textInputView.lineEndings = newValue
         }
     }
+    /// If this flag is enable the TextView will present the edit menu when select a range 
+    public var showEditMenuOnSelectRange: Bool = true
```

Comment:
Can we rename this to make it clear that it's related to highlighted ranges? Something like the following, maybe. It's long and boring but it's descriptive 😄

```suggestion
    public var showMenuAfterNavigatingToHighlightedRange = true
```